service_mesh/v2x/ossm-traffic-manage.adoc

```diff
@@ -48,7 +48,7 @@ Starting with {SMProductShortName} 2.5, automatic routes are disabled by default
 {SMProductName} creates the route with the subdomain, but {product-title} must be configured to enable it. Subdomains, for example `*.domain.com`, are supported, but not by default. Configure an {product-title} wildcard policy before configuring a wildcard host gateway.
 
 ifdef::openshift-enterprise[]
-For more information, see xref:../../networking/ingress-operator.adoc#using-wildcard-routes_configuring-ingress[Using wildcard routes].
+For more information, see xref:../../networking/networking_operators/ingress-operator.adoc#using-wildcard-routes[Using wildcard routes].
```

Comment:
All of the "using wildcard routes" links to "Ingress Operator in OpenShift Container Platform
"

Reply:
:heavy_check_mark: